editor/components/block-settings-menu/index.js

```diff
@@ -23,6 +23,11 @@ import BlockTransformations from './block-transformations';
 import SharedBlockSettings from './shared-block-settings';
 import UnknownConverter from './unknown-converter';
 
+const { Fill: PluginsBlockSettingsMenu, Slot: PluginsBlockSettingsMenuSlot } =
```

Comment:
This is how files with Slot/Fill look like: https://github.com/WordPress/gutenberg/blob/master/edit-post/components/sidebar/plugin-post-publish-panel/index.js.

Reply:
I personally don't like the pattern of assigning the `.Slot` into the Fill because we don't want to expose the Slots but I'm following the pattern now, we can change that later.